config/metar.php

```diff
@@ -2,6 +2,6 @@
 
 // Config for downloading METARs in the context of MSL and RPS
 return [
-    'vatsim_url' => env('VATSIM_METAR_URL', 'metar.vatsim.net'),
+    'vatsim_url' => env('VATSIM_METAR_URL', 'https://metar.vatsim.net'),
     'regional_url' => env('APP_REGIONAL_PRESSURES_URL', 'https://www.aviationweather.gov/adds/dataserver_current/httpparam?dataSource=metars&requesttype=retrieve&format=xml&hoursBeforeNow=3&mostRecentForEachStation=constraint&stationString=~gb&fields=raw_text,station_id'),
```

Comment:
Do you still need this `regional_url` given you remove it from the .env examples?

Reply:
f9ae0f19a16840f54d7f5e3a13ff1696ed1c1fac